activerecord/test/schema/postgresql_specific_schema.rb

```diff
@@ -141,4 +141,8 @@
     create_table(:measurements_concepcion, id: false, force: true,
                                            options: "PARTITION OF measurements FOR VALUES IN (2)")
   end
+
+  if supports_partial_index?
```

Comment:
`supports_index_include?`?

Reply:
Oof, thanks for catching this! I've fixed it.